README.md

```diff
@@ -10,4 +10,12 @@ and returns all messages from stdout stderr from cmd back on ws chanel to all cl
 example with php
```

Comment:
Vo vseobecnosti by som ti odporucil pouzivat nejaky lepsi linter.

Napr. my pouzivame toto:
https://github.com/keboola/keboola-as-code/blob/main/build/ci/golangci.yml

By default to ma zapnute nejake zakladne pravidla, my sme si zapli skoro vsetko, a potom sme vypli, co nedavalo zmysel.

Dava to celkom pekne odporucania, ako napisat kod lepsie.